rpc_jobs/standard_job_premerge.yml

```diff
@@ -93,5 +93,5 @@
         library "rpc-gating-master"
       }}
       if (! common.isSkippable(skip_pattern, "{credentials}")) {{
-        common.stdJob("check", "{credentials}", "", "{wrappers}")
+        common.stdJob("check", "{credentials}", "", "{wrappers}", "")
```

Comment:
Should there be two extra parameters here instead of one?

Reply:
Does it matter if it doesn't work?